### PR TITLE
linkchecker and setup.py tidy up

### PR DESCRIPTION
**Pull request by cjmayo**
Back to the list of tidy ups, mainly removing Python 2 code.

There are quite a few so if we did them individually we would be here quite a while. I'm trying to group them into themes of a small number of commits that don't overlap. Let me know if it doesn't work and there are too many (or too few...)

### CODE REVIEW

File: linkchecker [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python -Ru
+#!/usr/bin/python -u
```
Comment (mgedmin) [inline linkchecker:1]: Better make this
```suggestion
#!/usr/bin/python3 -u
```

File: setup.py [hunk 2]
```diff
@@ -373,19 +373,7 @@ def get_file_list (self):
...
-        'linkcheck.checker',
-        'linkcheck.configuration',
-        'linkcheck.director',
-        'linkcheck.htmlutil',
-        'linkcheck.logger',
-        'linkcheck.network',
-        'linkcheck.parser',
-        'linkcheck.plugins',
-    ],
+    packages = find_packages(include=["linkcheck", "linkcheck.*"]),
```
Comment (mgedmin) [inline setup.py:376]: :100:

File: linkchecker (hunk 1 above)
Comment (cjmayo) [reply linkchecker]: Thanks, I do forget not everyone has been using Python 3 as python.

File: setup.py (hunk 2 above)
Comment (cjmayo) [reply setup.py:376]: Someone's bright idea.

**Pull request merged**